advocacy_docs/migrating/oracle/oracle_epas_comparison/non_relational_data.mdx

```diff
@@ -9,6 +9,7 @@ title: "Comparison of nonrelational data support"
 | JSON support               | Yes <br/> Text based         | Yes <br/> Text and high-performance binary based |
 | Key-value store            | NoSQL database               | Yes                           |
 | Support for XML namespaces, DOM, XQuery, SQL/XML, and XSLT | XML DB                        | No                            |
+| Support for XMLDOM API package | Yes   | Yes  | 
```

Comment:
I don't think we need a new separate row in the table for this. Instead, I think we should change the "No" in the row above ("Support for XML namespaces, DOM, XQuery, SQL/XML, and XSLT") to

> Partial
> XML DOM support is provided via the DBMS_XMLDOM package